examples/azure-functions-timer-trigger/azure-functions-timer-trigger.bal

```diff
@@ -0,0 +1,14 @@
+import ballerinax/azure.functions;
```

Comment:
incorrect import order. https://learn-ballerina.github.io/best_practices/format_the_code.html#order-of-the-imports-in-a-bal-file

Reply:
Fixed.